CHANGELOG.md

```diff
@@ -1,6 +1,23 @@
 CHANGELOG
 =========
 
+2.10.2
+------
+**ENHANCEMENTS**
+
+- Improve cluster config validation by using cluster target AMI when invoking RunInstances in dryrun mode
+
+**CHANGES**
+
+- Make `key_name` parameter optional to support cluster configurations without a key pair. 
+- Remove support for Python 3.4
+- Root volume size increased from 25GB to 35GB on all AMIs. Minimum root volume size is now 35GB.
```

Comment:
We are probably not doing this

Reply:
Right; removed changelog lines not belonging to this commit